### DISCUSSION TITLE
How to change the status screens' colouring.

### DISCUSSION THREAD

**Discussion by TheWeirdCoder** [TownyAdvanced/Towny#6442] [Q&A] [answered]
Hey so i wanted to know how i can change the /t menu from

![image](https://user-images.githubusercontent.com/116373326/213327088-4217f97d-6a8b-42a0-a28b-fb22358fd048.png)

to

![image](https://user-images.githubusercontent.com/116373326/213327044-f605285f-ff35-41c5-bfe6-38ba45363f99.png)

thanks in advance <3

**Comment by LlmDl** [comment, marked as answer]
The towny\settings\lang\override\global.yml has the formatting strings for the status screens pre-filled but commented out. Check it out.
